functions/Backup-DbaDatabase.ps1

```diff
@@ -101,7 +101,11 @@ function Backup-DbaDatabase {
 
     .PARAMETER AzureCredential
         The name of the credential on the SQL instance that can write to the AzureBaseUrl, only needed if using Storage access keys
+<<<<<<< HEAD
+        If using SAS credentials, the command will look for a credential with a name matching the AzureBaseUrl. Setting this parameter will set number of files to 1 as striped sets are not supported
+=======
         If using SAS credentials, the command will look for a credential with a name matching the AzureBaseUrl. As page blobs are used with this option we force the number of files to 1 and ignore any value passed in for BlockSize or MaxTransferSize
+>>>>>>> development
```

Comment:
Seems merge conflict headers got committed into the file....had this happen with VS Code before.

Reply:
Yep, it's broken a load of stuff elsewhere (including casing as you point out below).

Will revert the merge and then sort everything else out.